Windows 365 Gateway IP Lookup/Get-AVDW365Gateways.ps1

```diff
@@ -98,26 +115,76 @@ function invoke-download {
         $count = $count + 1
         $index = $subnet.IndexOf('/')
         if ($count -ne $subnets.count) {
-        ($subnet.Substring(0, $index)) + ',' | out-file $PSSCriptRoot\temp.csv -Append
+            if ($IP -eq $true) { ($subnet.Substring(0, $index)) + ',' | out-file $PSSCriptRoot\temp.csv -Append }
```

Comment:
What's the value of IP? Is it a string or Boolean? If boolean, IMO, "isIP" or something like that could indicate is a boolean would be better name.

Reply:
It is a boolean. All of the flagged variables are boolean. I see your point when looking at this from a code "readability" perspective, but I think using "isIP" as a variable name may be confusing to users who actually need to call these variables at command line. In this scenario, I think the existing names make sense.

Thoughts?